zyte_spider_templates_project/settings.py

```diff
@@ -46,3 +49,24 @@
         "style"
     ],
 }
+
+# spidermon
+SPIDERMON_ENABLED = True
+SPIDERMON_SPIDER_CLOSE_MONITORS = (
+    "zyte_spider_templates_project.monitors.CustomSpiderCloseMonitorSuite",
+)
+SPIDERMON_MAX_ERRORS = 0
+SPIDERMON_ADD_FIELD_COVERAGE = True
+SPIDERMON_FIELD_COVERAGE_RULES = {
+    "Product/url": 1.0,
+    "Product/name": 1.0,
+    "Product/metadata": 1.0,
+}
+SPIDERMON_ITEM_COUNT_INCREASE = 1  # At least 1 item every 5 minutes
```

Comment:
Feels redundant with the recommended [CLOSESPIDER_TIMEOUT_NO_ITEM](https://docs.scrapy.org/en/latest/topics/extensions.html#std-setting-CLOSESPIDER_TIMEOUT_NO_ITEM) value of 600 in the [setup instructions](https://zyte-spider-templates.readthedocs.io/en/latest/setup.html#configuration).

Reply:
@VMRuiz what do you think?